remix-gql-tailwind/app/components/UserProfile/UserProfile.stories.tsx

```diff
@@ -13,11 +13,11 @@ const exampleData = {
   login: 'tvanantwerp',
   name: 'Tom VanAntwerp',
   twitterUsername: 'tvanantwerp',
-  websiteUrl: 'https://tomvanantwerp.com',
+  websiteUrl: '//tomvanantwerp.com',
```

Comment:
Because GitHub isn't added the protocol to these user website links, we need the component to handle that. In this case, there should just be nothing to simulate what we expect the data from GitHub to look like.

```suggestions
  websiteUrl: 'tomvanantwerp.com',
```